backend/base.py

```diff
@@ -530,6 +530,11 @@ def goalsUpdate(): # pragma: no cover
         "email": current_user,
     }
     try:
+        current_user = get_jwt_identity()
```

Comment:
Does current_user need to be assigned inside the try block again? It seems to have already been done outside.

Reply:
oops